src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -37,7 +40,8 @@ export default class ServiceGraphPage extends React.Component<ServiceGraphPagePr
     super(props);
 
     this.state = {
-      summaryData: { summaryType: 'graph', summaryTarget: null }
+      summaryData: { summaryType: 'graph', summaryTarget: null },
+      hideLegend: true
```

Comment:
Please put `hideLegend` into Redux store. The `state` for _ServiceGraphPage_ has actually already been removed and _ServiceGraphPage_ is a stateless component once the PR https://github.com/kiali/kiali-ui/pull/354 is merged.